auslib/blobs/base.py

```diff
@@ -107,3 +113,25 @@ def processSpecialForceHosts(self, url, specialForceHosts):
             else:
                 url += '?force=1'
         return url
+
+    def getHeaderXML(self, updateQuery, update_type):
+        raise NotImplementedError()
+
+    def getFooterXML(self):
+        raise NotImplementedError()
+
+    def getInnerXML(self):
+        raise NotImplementedError()
+
+    def createXML(self, updateQuery, update_type, whitelistedDomains, specialForceHosts):
+        # ONLY USED IN TESTS
```

Comment:
Hm. I hadn't realized that this method would be dead after the client.py changes. Can you kill this, and change the tests to use getHeader/Inner/FooterXML instead?